e2e/tasks/doAllPriceUpdatesSepolia.js

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env node
```

Comment:
This is unnecessary because `doPriceUpdate` for market that does not exist is a noop and ok to do. So global `doAllPriceUpdates` should work fine for any market (that exists or does not exist)